File: LeetCode/38.Count_And_Say/main.py
#!/usr/bin/python

from solution import Solution

# Unit Test
t1 = 1
t2 = 4
s = Solution()
if (s.countAndSay(t1)=="1"):
    print("Pass")
else:
    print("Test 1 Failed")
if (s.countAndSay(t2) == "1211"):
    print("Pass")
else:
    print("Test 2 Failed")

File: LeetCode/38.Count_And_Say/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    int t1 = 1;
    int t2 = 4;
    if (s.countAndSay(t1) == "1") { cout << "Pass" << endl;}
    else { cout << "Failed 1 - t1" << endl;}
    if (s.countAndSay(t2) == "1211") { cout << "Pass" << endl;}
    else { cout << "Failed 2 - t2" << endl;}
    return 0;
}